main.py

```diff
@@ -0,0 +1,8 @@
+import numpy as np
+from patblib import Path
+
+
```

Comment:
Oh! And you should add an `if __name__ == "__main__"` statement. This is good practice to have in executable files (i.e. files that you'll run and not just pull classes or functions from)

Reply:
I added this here f590e5b17071a60ad00fb17b8f24b3cfb67b0289